### DISCUSSION TITLE
Is this a typo?

### DISCUSSION THREAD

**Discussion by bluebabyxp** [NVIDIA/cutlass#1621] [General]
why DstRef is using srcEngine as template arg?

max_common_layout(Tensor<SrcEngine,SrcLayout> const& a,
                  Tensor<DstEngine,DstLayout> const& b)
{
  using SrcType = typename Tensor<SrcEngine,SrcLayout>::value_type;
  using DstType = typename Tensor<DstEngine,DstLayout>::value_type;
  using SrcRef  = typename Tensor<SrcEngine,SrcLayout>::reference;
  using DstRef  = typename Tensor<SrcEngine,SrcLayout>::reference;
}

**Comment by thakkarV** [comment]
yes, typo. CC @ccecka
